qa/rolling-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -5,38 +5,56 @@
 
 package org.opensearch.knn.bwc;
 
-import static org.opensearch.knn.TestUtils.*;
+import static org.opensearch.knn.TestUtils.NODES_BWC_CLUSTER;
 
 public class IndexingIT extends AbstractRollingUpgradeTestCase {
     private static final String TEST_FIELD = "test-field";
     private static final int DIMENSIONS = 5;
+    private int docId = 0;
     private static final int K = 5;
-    private static final int ADD_DOCS_CNT = 10;
+    private static final int NUM_DOCS = 10;
+    private int queryCount = 0;
 
-    public void testKnnDefaultIndexSettings() throws Exception {
+    public void testKNNDefaultIndexSettings() throws Exception {
         waitForClusterHealthGreen(NODES_BWC_CLUSTER);
         switch (getClusterType()) {
             case OLD:
                 createKnnIndex(testIndex, getKNNDefaultIndexSettings(), createKnnIndexMapping(TEST_FIELD, DIMENSIONS));
-                addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, 0, ADD_DOCS_CNT);
+                addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, docId, NUM_DOCS);
                 break;
             case MIXED:
                 if (isFirstMixedRound()) {
-                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, 10, K);
-                    addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, 10, ADD_DOCS_CNT);
-                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, 20, K);
+                    queryCount = NUM_DOCS;
+                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, queryCount, K);
+
+                    docId = NUM_DOCS;
+                    addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, docId, NUM_DOCS);
+
+                    queryCount = 2 * NUM_DOCS;
+                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, queryCount, K);
                 } else {
-                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, 20, K);
-                    addKNNDocs(testIndex, TEST_FIELD, DIMENSIONS, 20, ADD_DOCS_CNT);
-                    validateKNNSearch(testIndex, TEST_FIELD, DIMENSIONS, 30, K);
+                    queryCount = 2 * NUM_DOCS;
```

Comment:
Why are we setting queryCount to 2 * NUM_DOCS?

Reply:
May be it is not an appropriate name. I have updated the variable name. We are using it to generate the query vector